src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -1,23 +1,36 @@
 
-from typing import Any, Tuple
+from typing import Any, Tuple, IO
 import ipaddress
+import tempfile
+import os
+import logging
 
 from datetime import datetime, timedelta
 from cryptography import x509
 from cryptography.x509.oid import NameOID
 from cryptography.hazmat.primitives.asymmetric import rsa
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.backends import default_backend
+from OpenSSL import crypto, SSL
```

Comment:
both of them are used by the `def verify_tls(crt, key)` which I moved from` mgr_util.py` to the new `ssl_cert_utils.py` (I think this function makes more sense in the new location rather than the mgr util).

Reply:
okay, as said in the other thread, I don't really think these functions should be within cephadm and should stay at the mgr level, but I guess whether or not we import these libraries here is more dependent on that discussion than being its own issue